File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
 
 {% endtabs %}
 
+## Recreate Nested Metafile
+
+This setting allows you to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
```

Comment:
This setting allows you to regenerate the nested EMF images in the PowerPoint presentation document during PDF conversion.

Reply:
Feedback addressed